.travis.yml

```diff
@@ -16,13 +16,18 @@ before_install:
 - sudo apt-get install -y libjson-c-dev libcurl4-gnutls-dev libsystemd-journal-dev
 - sudo apt-get install -y clang
 - sudo apt-get install -y lcov
+- sudo apt-get install -y rpm
 - gem install coveralls-lcov
+- mkdir -p ~/rpmbuild/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
 before_script:
 - autoreconf -i -f
 - $CLANG_SCAN_BUILD ./configure --disable-silent-rules $CONFIGURE_ARGS
 script:
+- ./configure --prefix=/usr
 - $CLANG_SCAN_BUILD make
 - $CLANG_SCAN_BUILD make check
+- make distcheck
```

Comment:
Did I get it right, that this still fails, even without `$CLANG_SCAN_BUILD`, because of the `.gcno` files? If so, then I was mistaken, and it's the coveralls, who asks gcc to create those files. A good fix would be to add removal of `.gcno` files from all directories where object files are created, to the `clean` target. I guess that can be done by adding the `clean-local` target to the affected Makefile.am`s (with a command to remove them) as described in https://www.gnu.org/software/automake/manual/automake.html#Extending